internal/ociutils/ociutils.go

```diff
@@ -0,0 +1,255 @@
+package ociutils
```

Comment:
Can we find a more descriptive name? `ociwrapper` or `oci`? 

>Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them.

https://go.dev/blog/package-names

Reply:
I changed it to `ociwrapper` as `oci` was too generic